uint8 PACKET_TYPE_NONE = 0
uint8 PACKET_TYPE_TEST = 1
uint8 PACKET_TYPE_NAMED_STRING = 11
uint8 PACKET_TYPE_NAMED_INT = 12
uint8 PACKET_TYPE_NAMED_FLOAT = 13
uint8 PACKET_TYPE_SENSOR_ENV_III = 21
uint8 PACKET_TYPE_SENSOR_UNITV2_PERSON_COUNTER = 22
uint8 PACKET_TYPE_EMERGENCY = 31
uint8 PACKET_TYPE_TASK_DISPATCHER = 32
uint8 PACKET_TYPE_TASK_RESULT = 33
uint8 PACKET_TYPE_TASK_RECEIVED = 34
uint8 PACKET_TYPE_DEVICE_MESSAGE_BOARD_META = 41
uint8 PACKET_TYPE_DEVICE_MESSAGE_BOARD_DATA = 42
uint8 PACKET_TYPE_META = 81
uint8 PACKET_TYPE_DATA = 82

uint8[] mac_address # Must be 6
uint8[] data
